### PR TITLE
Milestone four

### PR DESCRIPTION
**Pull request by RoyNyaga**
Created a logic folder and included a calculator.js file which contains a calculator module. Also created operator.js file which contains operator function.

### CODE REVIEW

File: src/logic/calculator.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import operator from './operator'
+
+const calculate = (() => {
+  const calculate = (calculatorObject, buttonName) => {
```
Comment (azdrenymeri) [inline src/logic/calculator.js]: You need to cover cases when `buttonName` equals to `"="` or `"."` or `"AC"`.

File: src/logic/operator.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+const operator = (numberOne, numberTwo, operation) => {
+  const bigNumberOne = Big(numberOne)
+  const bigNumberTwo = Big(numberTwo)
+  switch (operation) {
+    case '+':
+      return bigNumberOne.plus(bigNumberTwo).toString()
+    case '-':
+      return bigNumberOne.minus(bigNumberTwo).toString()
+    case '/':
```
Comment (azdrenymeri) [inline src/logic/operator.js]: If `numberOne` is zero and `numberTwo` is zero this will throw a `DividedByZero` error.
Kindly handle this error

**Comment by RoyNyaga** [comment]
coverd cases for "AC", "=", and ".". Also included error message for none divisible by zero.

**Comment by RoyNyaga** [comment]
Yuuuupppppiiiiii thanks sir

**Pull request merged**